### DISCUSSION TITLE
In My Fastapi Service if, i add a      root_path="/api/app"         in Fastapi definition    Api's are not reachable from ingress and only simple health endpoint is reaching and swagger docs also not working [Fetch error response status is 404 /openapi.json]

### DISCUSSION THREAD

**Discussion by Karthik-codes-I** [fastapi/fastapi#11936] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
app = FastAPI(lifespan=lifespan, root_path="/api/app", docs_url=docs_url, redoc_url=None)
```

### Description

My Micro-service Proxy route is **"/api/app"**  and if i add **root_path** as **"/api/app"**  simple health endpoint is working fine and other api's are not reaching to server.

### Operating System

Linux

### Operating System Details

Ubuntu 20.04

### FastAPI Version

fastapi==0.110.2

### Pydantic Version

pydantic==1.9.0

### Python Version

python:3.9.11

**Comment by estebanx64** [comment, marked as answer]
Hi!

Can you please provide the full details so we can determine what is happening?

* What is your ingress configuration? Are you using a k8s cluster? ingress nginx? traefik any other stuff?
* The full example of the piece of code of FastAPI with the docs_url.

Moreover, if you haven't seen this section it will be good to glance at it.
https://fastapi.tiangolo.com/advanced/behind-a-proxy/?h=root#disable-automatic-server-from-root_path
